### ISSUE TITLE
What Should 1.0.0 Contain

### ISSUE THREAD

**Issue by StrangeWill** [RoushTech/RollbarDotNet#36]
Trying to run with solid Semver here...

* Separate libraries for core Rollbar and Rollbar support for web apps, minimum requirements for the former.
* netstandard2.0? I can't really support Rollbar to the level that I'd like until that comes out.

**Comment by NewtonRocha** [comment]
I think that will be nice, if we have a deploy support ! that's a awesome feature =D

**Comment by StrangeWill** [comment]
@NewtonRocha got a use case for embedding the deploy call in your application? Usually that's done by your CI/Deploy servers (TeamCity/CircleCi/Octopus Deploy/etc.).

It wouldn't be hard to add it if it'll actually get used.

**Comment by NewtonRocha** [comment]
Yes, but will be awesome if we can put that function on publish, to auto generate deploy for the guys who doenst has CI or doenst want to depend on a CI to generate a deploy on rollbar, like when you build a docker image version, i want that every instance using that image use the same "deploy". (sorry for my poor English, i'm a little rusty )
